flowzone.yml

```diff
@@ -1890,12 +1908,18 @@ jobs:
 
   # check if the repository has a package.json file and which engine versions are supported
   is_npm:
-    name: Check NodeJS versions
+    name: request branch for pull requestJS versions
```

Comment:
```suggestion
    name: request branch for pull request JS versions
```

Reply:
That must have been a copy-paste error, I didn't even mean to update that job name? Weird.

Reply:
I thought it was weird too, but I assumed you had some reason? :p

Reply:
That's very nice of you